### DISCUSSION TITLE
any ways to suppress output channel caused by hbargraph?

### DISCUSSION THREAD

**Discussion by oldstar84** [grame-cncm/faust#1091] [Q&A]
hi,
I got 2 channels for stereo and another one caused by hbargraph. the first 2 channels do what they should and the hbargraph also. Is there a way to suppress the third output channel which was added while using hbargraph?

It is not possible to terminate like this 
`: _,_,!: `
because hbargraph don't show up then.

Thanks for any tips!

**Comment by sletz** [comment]
Look at the [attach primitive](https://faustdoc.grame.fr/manual/syntax/#attach-primitive).
